TermTk/TTkCore/signal.py

```diff
@@ -88,7 +154,7 @@ def __init__(self, *args, **kwargs):
         self._connected_slots = []
         _pyTTkSignal_obj._signals.append(self)
 
-    def connect(self, slot):
+    def connect(self, slot, use_weak_ref=False):
```

Comment:
We need to think about having the "weak ref" true by default,
I am not sure what it is going to happen if connected to lambda or nedted functions. 
For the naming convention I tried to use the camel case although not recommended by PEP8
because I started coding it following the QT api structure